functions/core.php

```diff
@@ -230,6 +230,38 @@ function hmbkp_update() {
 		);
 
 		array_map( 'delete_transient', $transients );
+
+		// Clear duplicate schedules on multisite
+		if ( is_multisite() ) {
+
+			// get current blogs from DB
+			$blogs = wp_get_sites();
+
+			foreach ( $blogs as $blog ) {
+
+				switch_to_blog( get_current_blog_id() );
+
+				if ( is_main_site( get_current_blog_id() ) ) {
+					continue;
+				}
+
+				global $wpdb;
+
+				// Get the schedule options
+				$schedules = $wpdb->get_col( $wpdb->prepare( "SELECT option_name FROM $wpdb->options WHERE option_name LIKE %s", 'hmbkp_schedule_%' ) );
```

Comment:
You can, but you need to include the file that contains the function directly as the plugin isn't loaded when `uninstall.php` is run.